metrics_utility/test/ccspv_reports/test_consistent_indirectly_managed_nodes_output.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+
+import openpyxl
+import pandas
+import pytest
+
+from conftest import temporary_env
+
+from metrics_utility.management.commands.build_report import Command
+
+
+env_vars = {
+    'METRICS_UTILITY_PRICE_PER_NODE': '11.55',
+    'METRICS_UTILITY_REPORT_RHN_LOGIN': 'test_login',
+    'METRICS_UTILITY_SHIP_PATH': './metrics_utility/test/test_data',
+    'METRICS_UTILITY_REPORT_END_USER_COMPANY_NAME': 'Customer A',
+    'METRICS_UTILITY_REPORT_END_USER_STATE': 'TX',
+    'METRICS_UTILITY_REPORT_SKU_DESCRIPTION': 'EX: Red Hat Ansible Automation Platform, Full Support (1 Managed Node, Dedicated, Monthly)',
+    'METRICS_UTILITY_REPORT_H1_HEADING': 'CCSP NA Direct Reporting Template',
+    'METRICS_UTILITY_REPORT_END_USER_CITY': 'Springfield',
+    'METRICS_UTILITY_REPORT_PO_NUMBER': '123',
+    'METRICS_UTILITY_SHIP_TARGET': 'directory',
+    'METRICS_UTILITY_REPORT_END_USER_COUNTRY': 'US',
+    'METRICS_UTILITY_REPORT_COMPANY_NAME': 'Partner A',
+    'METRICS_UTILITY_REPORT_SKU': 'MCT3752MO',
+    'METRICS_UTILITY_REPORT_EMAIL': 'email@email.com',
+    'METRICS_UTILITY_REPORT_TYPE': 'CCSPv2',
+}
+
+indirect_nodes_and_managed_by_org_file_path = './metrics_utility/test/test_data/reports/2025/02/CCSPv2-2025-02-13--2025-02-13.xlsx'
+indirectly_managed_nodes_file_path = './metrics_utility/test/test_data/reports/2025/02/CCSPv2-2025-02-13--2025-02-13.xlsx'
+date_today = datetime.now().strftime('%b %d, %Y')
```

Comment:
Both `indirect_nodes_and_managed_by_org_file_path` and `indirectly_managed_nodes_file_path` are the same file, maybe this doesn't have to be 2 vars?

(Or, if it does, only one of them is currently being cleaned up)

And `date_today` is unused, remove? :)